Compilation error with recent libQGLViewer>=2.6.0

After a discussion on the CGAL-develop mailing list, where I asked developers to try the RC of CGAL-4.6.1...

A compilation error in `demo/Mesh_3/` with g++ 5.1.0:
```
Friday 26 June 2015 15:24:56 Alexander Kobel wrote :
> ========================================================================
> == Arch up-to-date, x86_64 (in chroot of the above Debian), g++ 5.1.0
> == RS3 manually installed, everything else from the repos
> == -DWITH_{GMPXX,MPFI,RS,RS3,NTL,demos,examples}=On
> == Compiled all but one from demos and all examples successfully,
> == except those requiring ESBTL, OpenMesh, and/or Qt3.
> ========================================================================
> .../demo/Mesh_3/Volume_planes_plugin.cpp: In member function ‘void
> Plane_slider::updateValue()’:
> .../demo/Mesh_3/Volume_planes_plugin.cpp:120:31: error:invalid
> initialization of non-const reference of type ‘qreal& {aka double&}’
> from an rvalue of type ‘qreal {aka double}’
>       frame->getPosition(a, b, c);
>                                 ^
> 
> In file included from /usr/include/QGLViewer/keyFrameInterpolator.h:31:0,
>                   from /usr/include/QGLViewer/camera.h:26,
>                   from /usr/include/QGLViewer/qglviewer.h:26,
>                   from
> /home/akobel/CGAL-4.6.1/demo/Mesh_3/Volume_plane.h:15,
>                   from
> /home/akobel/CGAL-4.6.1/demo/Mesh_3/Volume_planes_plugin.cpp:5:
> /usr/include/QGLViewer/frame.h:188:7: note:   initializing argument 1 of
> ‘void qglviewer::Frame::getPosition(qreal&, qreal&, qreal&) const’
>    void getPosition(qreal& x, qreal& y, qreal& z) const;
>         ^
```
and also the same compilation bug with clang++ 3.6.1:
```
> ========================================================================
> == Arch up-to-date, x86_64 (in chroot of Debian), clang++ 3.6.1
> == RS3 manually installed, everything else from the repos
> == -DWITH_{GMPXX,MPFI,RS,RS3,NTL,demos,examples}=On
> == -DCMAKE_BUILD_TYPE=Debug
> == Compiled all but one from demos and all examples successfully,
> == except those requiring ESBTL, OpenMesh, and/or Qt3.
> ========================================================================
> /home/akobel/CGAL-4.6.1/demo/Mesh_3/Volume_planes_plugin.cpp:120:24: error:
>        non-const lvalue reference to type 'qreal' (aka 'double') cannot
> bind to a value
>        of unrelated type 'float'
>      frame->getPosition(a, b, c);
>                         ^
> /usr/include/QGLViewer/frame.h:188:26: note: passing argument to
> parameter 'x' here
>          void getPosition(qreal& x, qreal& y, qreal& z) const;
>                                  ^
```

--- a/Mesh_3/demo/Mesh_3/Volume_planes_plugin.cpp
+++ b/Mesh_3/demo/Mesh_3/Volume_planes_plugin.cpp
@@ -116,10 +116,15 @@ public:
 
 public slots:
   void updateValue() {
-    float a, b, c;
+#if QGLVIEWER_VERSION >= 0x020600
+    typedef qreal qglviewer_real;
+#else // QGLViewer < 2.6.0
+    typedef float qglviewer_real;
+#endif // QGLViewer < 2.6.0
+    qglviewer_real a, b, c;
     frame->getPosition(a, b, c);
-    float sum1 = a + b + c;
-    float sum2 = v.x + v.y + v.z;
+    float sum1 = float(a + b + c);
+    float sum2 = float(v.x + v.y + v.z);
     sum1 /= sum2;
     setValue(sum1 * scale);
   }

